Sources/Testing/Running/XCTestScaffold.swift

```diff
@@ -76,7 +76,7 @@ extension XCTIssue {
 /// ## See Also
 ///
 /// - <doc:TemporaryGettingStarted>
-#if SWIFT_PM_SUPPORTS_SWIFT_TESTING
+#if compiler(>=5.11)
```

Comment:
Should we also be converting all of these to `6.0` soon?

Reply:
Probably. I want to wait a _little_ longer for people to get off 5.11 toolchains though.